test/e2e/network/ingress.go

```diff
@@ -361,6 +362,38 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Unexpected error while waiting for ingress: %v", err))
 		})
 
+		It("should create ingress with backside re-encryption", func() {
```

Comment:
Probably not at this point. I can already see the implementations of different ingress controllers diverged --- nginx put the configuration on ingress object ([doc](https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/annotations.md#secure-backends)) while gce put it on service. Trying to avoid heavy lifting before code freeze...

Reply:
hmm ok. Thats unfortunate.
Will need to figure out a way to run this test with kubemci as well.
Thats the only reason why I suggested incorporating this in conformance tests.

Reply:
> Will need to figure out a way to run this test with kubemci as well.

I'm thinking we could probably add an abstraction layer to generate GCE specific conformance test. Haven't thought deeply into that though.